woocommerce/woocommerce-ios
# Product Description AI: Update prompt to be clearer to fix issue detecting language

Closes: #10103 


## Description

The prompt we are currently using for product description needs improvements to make AI better understand our intention. This PR updates the prompt in an attempt to enhance language detection and ensure the backticks are not included in the response.

## Testing instructions

- Log in to a published WPCom site.
- Select an existing product or create a new one.
- Select Write with AI in the description row.
- On the description AI sheet, select Write with AI. The response should be in the same language as the product title and features.
- Tap Regenerate and make sure that new responses are still good.

## Screenshots

N/A

---
